packages/ra-data-graphql-simple/package.json

```diff
@@ -33,20 +33,25 @@
         "watch": "tsc --outDir dist/esm --module es2015 --watch"
     },
     "dependencies": {
-        "@apollo/client": "^3.3.19",
+        "@apollo/client": "^3.6.9",
```

Comment:
Please rebase on the `next` branch. This upgrade of dependencies to a minor version must be released as a minor version - not as a bugfix version.